src/monattack.h

```diff
@@ -32,6 +33,7 @@ bool vine( monster *z );
 bool spit_sap( monster *z );
 bool triffid_heartbeat( monster *z );
 bool fungus( monster *z );            // Generic fungal spore-launch
+bool fungus_corporate( monster *z );   // BUY OUR PRODUCT. Used by Crazy Cataclysm; spawns SpOreos(tm).
```

Comment:
This comment makes line too long - either move it to separate line or update its text.